Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.9-alpine
+
+RUN touch /config.json
+
+RUN adduser --disabled-password --home=/app syncmymoodle
```

Comment:
Why is the additional user necessary?

Reply:
isn't the idea to have a custom user to have distinct owners (especially non-root) on the host to make use of proper permissions?
if so, can this container also choose an uncommon uid?

Reply:
The prefered way is to have the user have the same uid as the host user. As this is different between systems it should not be part of the Dockerfile but rather inside of the compose file. Users inside the Dockerfile themselves are primarily for applications like apache/nginx which drop privileges to another user which is only allowed to read static files.